### DISCUSSION TITLE
Voyager navigation

### DISCUSSION THREAD

**Discussion by forceindia712** [adrielcafe/voyager#257] [Q&A]
Hello,
Is it possible in Voyager to listen for a transition to the next or previous view?

I would like to display the dialogue when I try to move, but I don't know how to do it using Voyager, and how to block the transition until the dialogue is closed?

**Comment by wpoosanguansit** [comment]
Hi,

I am not quite sure if this would fit what you are looking for.

```
LifecycleEffect(
            onStarted = {
                //Display your Dialog
            },
            onDisposed = {
 
            }
        )
```

This is defined within the Content() of your screen.
